### PR TITLE
Sam/database service

### PR DESCRIPTION
**Pull request by SamanthaVS-BBD**
added new check user end point 
added inserting doc in db logic

### CODE REVIEW

File: Api/appsettings.json [hunk 1]
```diff
@@ -13,5 +13,9 @@
       }
     ]
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "DatebaseOptions": {
+    "ConnectionString": "Host=mydbinstance.chylgcp0m93v.eu-west-1.rds.amazonaws.com;Port=5432;Database=SpectacularDocumentationDB;Username=adminuser;Password=randompassword",
```
Comment (seanvanwyk-bbd) [inline Api/appsettings.json]: !! Connection string!!

Comment (SamanthaVS-BBD) [reply Api/appsettings.json]: ohh shit right thank you

File: OAuth/OAuthCLI/OAuth.cs [hunk 2]
```diff
@@ -12,7 +12,8 @@ public class OAuth
     public OAuth(HttpClient client)
     {
         this.client = client;
-        var envClientID = Environment.GetEnvironmentVariable("GithubClientID") ?? throw new Exception("Environment Variable for Github Client ID not set!");
+        //var envClientID = Environment.GetEnvironmentVariable("GithubClientID") ?? throw new Exception("Environment Variable for Github Client ID not set!");
+        var envClientID = "Iv1.be664de67499daba";
```
Comment (seanvanwyk-bbd) [inline OAuth/OAuthCLI/OAuth.cs]: And clientID. Though I know we weren't sure about these.

**Review by seanvanwyk-bbd — CHANGES_REQUESTED** [review]
Didn't see anything else problematic, just the variables. Very nice stuff done here though, thank you!

**Pull request merged**